TLM/TLM/Patch/_VehicleAI/UpdatePathTargetPositionsPatch.cs

```diff
@@ -777,7 +777,7 @@ public class UpdatePathTargetPositionsPatch {
                         if (logLogic) {
                             Log._Debug(
                                 $"CustomVehicle.CustomUpdatePathTargetPositions({vehicleID}): " +
-                                "Next next path position needs to be calculated. Used triple-calculation. " +
+                                "Next path position needs to be calculated. Used triple-calculation. " +
```

Comment:
That additional "next" is important here. Means the next after current next = 2nd next position